actors/builtin/miner/miner_actor.go

```diff
@@ -942,8 +942,8 @@ func (a Actor) ProveCommitAggregate(rt Runtime, params *ProveCommitAggregatePara
 		svis = append(svis, svi)
 	}
 
-	sealProof := precommits[0].Info.SealProof
 	builtin.RequireState(rt, len(precommits) > 0, "bitfield non-empty but zero precommits read from state")
+	sealProof := precommits[0].Info.SealProof
```

Comment:
Note this should be strictly impossible since GetAllPrecommittedSectors should error and cause an abort before reaching here so I can't test.